src/applications/static-pages/vre-chapter31/Chapter31CTA.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { toggleValues } from 'platform/site-wide/feature-toggles/selectors';
+import FEATURE_FLAG_NAMES from 'platform/utilities/feature-toggles/featureFlagNames';
+import LoadingIndicator from '@department-of-veterans-affairs/formation-react/LoadingIndicator';
+
+import { buildChapter31Content } from './buildChapter31Content';
+
+const Chapter31CTA = props => {
+  let content;
+  const currentURL = window.location.href.split('/');
```

Comment:
It might be more contextual to put this in `buildChapter31Content` and just let that component handle all interaction with the window.

Reply:
I wanted to have the buildChapter31Content function only be concerned with returning content, no logic beyond that. You think that is the wrong way to go?

Reply:
Not sure tbh. I do think it might make things a little safer? What if URLs change in the future, or currentURL[5] or currentURL[6] don't always equal the expected values? Maybe making `buildChapter31Content` responsible for parsing the URL and providing a default fallback to handle anything expected might be a little safer.

Reply:
This will only be used for a week or so while we do the progressive rollout and only on the specific pages in Drupal that lead to the CH31 form. I figured since it is only being used on these few URLs and only for a week we should be ok. Honestly, if the URLs change during that week then we got other problems :-). I can move the URL stuff if you feel strongly about it, I am not trying to be recalcitrant, but I wanted to make sure you had the full context.